modules/ROOT/pages/_partials/include-general-config.adoc

```diff
@@ -61,6 +61,9 @@ After you complete your configuration, click *Save Changes*.
 
 To exit the configuration and return to the dashboard, click the *X*.
 
+[NOTE]
+When you create an application panel using a JVM metric, the query that supports the panel uses only the worker-id for the app. When you modify the panel at a later time, the resource fields are populated with the query content. Because some apps share a worker-id, the resource that appears in the selector may appear to be a different resource from the resource you selected originally. However, all resources share the same worker, so the query has the correct information despite the selector displaying a different resource name.
```

Comment:
```suggestion
When you create an application panel using a JVM metric, the query that supports the panel uses only the worker-id for the app. When you modify the panel at a later time, the resource fields are populated with the query content. Because some apps share a worker-id, the resource that appears in the selector might appear to be a different resource from the resource you selected originally. However, all resources share the same worker, so the query has the correct information despite the selector displaying a different resource name.
```